dcs/mapping.py

```diff
@@ -97,6 +97,11 @@ class Vector2:
     x: float
     y: float
 
+@dataclass
+class WWIIFollowBigFormationOrder:
+    x: int # distance
```

Comment:
The comments are better names. DCS might call them x/y/z, but it's a damned confusing API. The pydcs API doesn't need to exactly match the miz structure.

Include units in the name when you rename them. `distance_meters` or whatever this happens to be.

`int` is also incorrect. Lua doesn't have integers, only floats.